t/server/controller/contributor.t

```diff
@@ -0,0 +1,62 @@
+use strict;
+use warnings;
+
+use Cpanel::JSON::XS qw( decode_json );
+use MetaCPAN::Server::Test;
+use MetaCPAN::TestServer;
+use Test::More;
+
+my $server = MetaCPAN::TestServer->new;
+$server->index_packages;
```

Comment:
copy-pasted from the package test... not sure why it's there either - removing both.